userspace/personality_test/tock_shims.c

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC
+// Copyright 2018 Google LLC
```

Comment:
Merge error (copyright year)?

Reply:
Yeah, I copied over a new version of a file that had been first created in 2018. I'll fix.